apps/prairielearn/src/lib/group-update.ts

```diff
@@ -181,7 +181,8 @@ export async function randomGroups(
             const usersToMove = userGroups
               .filter((group) => group.length > min_group_size)
               .slice(smallGroup.length - min_group_size) // This will be negative (get the last n groups)
-              .map((group) => group.pop() as string);
+              // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Very funny that https://typescript-eslint.io/rules/non-nullable-type-assertion-style/ and https://typescript-eslint.io/rules/no-non-null-assertion/ seem to have completely contradictory opinions about this. Does it actually make sense to enable both of them?